unit-tests/src/test/java/org/eclipse/collections/impl/multimap/AbstractMutableMultimapTestCase.java

```diff
@@ -132,6 +135,33 @@ public void testPutAll()
     }
 
     @Test
+    public void testPutAllPairs()
+    {
+        MutableMultimap<Integer, String> multimap1 = this.newMultimapWithKeysValues(1, "One", 2, "2");
+        MutableList<Pair<Integer, String>> pairs1 = Lists.mutable.of(Tuples.pair(1, "One"), Tuples.pair(2, "Two"), Tuples.pair(3, "Three"));
+        Assert.assertTrue(multimap1.putAllPairs(pairs1));
+        MutableMultimap<Integer, String> expected1 = this.newMultimapWithKeysValues(1, "One", 1, "One", 2, "2", 2, "Two");
+        expected1.put(3, "Three");
+        Assert.assertEquals(expected1, multimap1);
```

Comment:
Yes that's right, it is because of the overloads going only up to 4 keys. I can make it consistent by creating an empty Multimap and then go on adding a key and value. Another option is to only use 4 keys, but I think there should be a test with duplicates.
Which one would you prefer? Or is there an option better than the two I mentioned?